### DISCUSSION TITLE
I am new here want to reset the battle so I can do it from scratch?

### DISCUSSION THREAD

**Discussion by nastanford** [cssbattle/forum#384] [General]
I am new here want to reset the battle so I can do it from scratch?  I did the battle very first few a long time ago I just[want to start it again.  Is there a way to do that?  or do I have to create a new email every time I come back if I want to start from scratch? I searched and figured the answer would be there but I didn't see it.

**Comment by chinchang** [comment]
Hey @nastanford ! I am afraid there is no way to reset the data. Creating a new account is the only way to start afresh. That said, I am curious to know why do you want to start from scratch and why not just continue on the old account itself and play the new targets?

**Comment by nastanford** [comment]
I'm focusing on revisiting initial tasks to strengthen memory recall.
Employing a spaced repetition technique, I review and practice
problem-solving without relying on solutions. Given my limited need for CSS
at work, this method helps me master its intricacies efficiently.

Thanks,

[image: photo]
*Nathan Stanford*
Programmer/Manager
p:302- <302-359-4498>359-4498 | ***@***.*** |
<http://www.facebook.com/Nathan.StanfordSr>
<http://www.linkedin.com/in/nathanstanford>
<http://twitter.com/nathanstanford>

   - My Author Page <http://www.amazon.com/author/nathanstanfordsr>
   - My Book "Loving Your Spouse"

   <http://www.amazon.com/Loving-Your-Spouse-Learning-journey/dp/148016948X/ref=la_B00BN8ZM9W_1_1?ie=UTF8&qid=1362402723&sr=1-1>
